### ISSUE TITLE
Error Reading long description during setup

### ISSUE THREAD

**Issue by Billcountry** [harlowja/fasteners#68]
I'm experiencing this error when setting up a google cloud dataflow pipeline.
Doing a `pip install -r requirements.txt` works when running this locally but once done from the setup scripts the error below occurs.

```log
Collecting cython (from -r requirements.txt (line 1))
  File was already downloaded /tmp/dataflow-requirements-cache/Cython-0.29.23.tar.gz
Collecting apache-beam[gcp]==2.29.0 (from -r requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/c5/ea/67ed679135ca9464579b95979defce31fb79ea0b880b920a495c70b440e2/apache-beam-2.29.0.zip (2.5MB)
  Saved /tmp/dataflow-requirements-cache/apache-beam-2.29.0.zip
Collecting elasticsearch==7.1.0 (from -r requirements.txt (line 4))
  File was already downloaded /tmp/dataflow-requirements-cache/elasticsearch-7.1.0.tar.gz
Collecting requests==2.25.1 (from -r requirements.txt (line 5))
  File was already downloaded /tmp/dataflow-requirements-cache/requests-2.25.1.tar.gz
Collecting urllib3<=1.25.11 (from -r requirements.txt (line 8))
  File was already downloaded /tmp/dataflow-requirements-cache/urllib3-1.25.11.tar.gz
Collecting avro-python3!=1.9.2,<1.10.0,>=1.8.1 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/avro-python3-1.9.2.1.tar.gz
Collecting crcmod<2.0,>=1.7 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/crcmod-1.7.tar.gz
Collecting dill<0.3.2,>=0.3.1.1 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/dill-0.3.1.1.tar.gz
Collecting fastavro<2,>=0.21.4 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/fastavro-1.4.1.tar.gz
Collecting future<1.0.0,>=0.18.2 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/future-0.18.2.tar.gz
Collecting grpcio<2,>=1.29.0 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/grpcio-1.38.0.tar.gz
Collecting hdfs<3.0.0,>=2.1.0 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/hdfs-2.6.0.tar.gz
Collecting httplib2<0.18.0,>=0.8 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/httplib2-0.17.4.tar.gz
Collecting numpy<1.21.0,>=1.14.3 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/numpy-1.19.5.zip
Collecting pymongo<4.0.0,>=3.8.0 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/pymongo-3.11.4.tar.gz
Collecting oauth2client<5,>=2.0.1 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/oauth2client-4.1.3.tar.gz
Collecting protobuf<4,>=3.12.2 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/protobuf-3.17.2.tar.gz
Collecting pyarrow<4.0.0,>=0.15.1 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/62/d3/a482d8a4039bf931ed6388308f0cc0541d0cab46f0bbff7c897a74f1c576/pyarrow-3.0.0.tar.gz (682kB)
  Saved /tmp/dataflow-requirements-cache/pyarrow-3.0.0.tar.gz
Collecting pydot<2,>=1.2.0 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/pydot-1.4.2.tar.gz
Collecting python-dateutil<3,>=2.8.0 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/python-dateutil-2.8.1.tar.gz
Collecting pytz>=2018.3 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/pytz-2021.1.tar.gz
Collecting typing-extensions<3.8.0,>=3.7.0 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/typing_extensions-3.7.4.3.tar.gz
Collecting dataclasses (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/1f/12/7919c5d8b9c497f9180db15ea8ead6499812ea8264a6ae18766d93c59fe5/dataclasses-0.8.tar.gz
  Saved /tmp/dataflow-requirements-cache/dataclasses-0.8.tar.gz
Collecting cachetools<5,>=3.1.0 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/cachetools-4.2.2.tar.gz
Collecting google-apitools<0.5.32,>=0.5.31 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/google-apitools-0.5.31.tar.gz
Collecting google-auth<2,>=1.18.0 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/google-auth-1.30.1.tar.gz
Collecting google-cloud-datastore<2,>=1.7.1 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/google-cloud-datastore-1.15.3.tar.gz
Collecting google-cloud-pubsub<2,>=0.39.0 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/google-cloud-pubsub-1.7.0.tar.gz
Collecting google-cloud-bigquery<2,>=1.6.0 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/google-cloud-bigquery-1.28.0.tar.gz
Collecting google-cloud-core<2,>=0.28.1 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/google-cloud-core-1.6.0.tar.gz
Collecting google-cloud-bigtable<2,>=0.31.1 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/google-cloud-bigtable-1.7.0.tar.gz
Collecting google-cloud-spanner<2,>=1.13.0 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/google-cloud-spanner-1.19.1.tar.gz
Collecting grpcio-gcp<1,>=0.2.2 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/grpcio-gcp-0.2.2.tar.gz
Collecting google-cloud-dlp<2,>=0.12.0 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/google-cloud-dlp-1.0.0.tar.gz
Collecting google-cloud-language<2,>=1.3.0 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/google-cloud-language-1.3.0.tar.gz
Collecting google-cloud-videointelligence<2,>=1.8.0 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/google-cloud-videointelligence-1.16.1.tar.gz
Collecting google-cloud-vision<2,>=0.38.0 (from apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/google-cloud-vision-1.0.0.tar.gz
Collecting chardet<5,>=3.0.2 (from requests==2.25.1->-r requirements.txt (line 5))
  File was already downloaded /tmp/dataflow-requirements-cache/chardet-4.0.0.tar.gz
Collecting idna<3,>=2.5 (from requests==2.25.1->-r requirements.txt (line 5))
  File was already downloaded /tmp/dataflow-requirements-cache/idna-2.10.tar.gz
Collecting certifi>=2017.4.17 (from requests==2.25.1->-r requirements.txt (line 5))
  File was already downloaded /tmp/dataflow-requirements-cache/certifi-2021.5.30.tar.gz
Collecting six>=1.5.2 (from grpcio<2,>=1.29.0->apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/six-1.16.0.tar.gz
Collecting docopt (from hdfs<3.0.0,>=2.1.0->apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/docopt-0.6.2.tar.gz
Collecting pyasn1>=0.1.7 (from oauth2client<5,>=2.0.1->apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/pyasn1-0.4.8.tar.gz
Collecting pyasn1-modules>=0.0.5 (from oauth2client<5,>=2.0.1->apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/pyasn1-modules-0.2.8.tar.gz
Collecting rsa>=3.1.4 (from oauth2client<5,>=2.0.1->apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/rsa-4.7.2.tar.gz
Collecting pyparsing>=2.1.4 (from pydot<2,>=1.2.0->apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/pyparsing-2.4.7.tar.gz
Collecting fasteners>=0.14 (from google-apitools<0.5.32,>=0.5.31->apache-beam[gcp]==2.29.0->-r requirements.txt (line 2))
  File was already downloaded /tmp/dataflow-requirements-cache/fasteners-0.16.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-0rcosqmh/fasteners/setup.py", line 25, in <module>
        long_description = readme.read()
      File "/usr/lib/python3.6/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xf0 in position 429: ordinal not in range(128)
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-0rcosqmh/fasteners/
```

**Comment by psarka** [comment]
I think I know how to fix it, give me a moment.

**Comment by psarka** [comment]
@Billcountry Published a 0.16.1 with a fix that I think will solve this. Give it a try!

**Comment by Billcountry** [comment]
@psarka Thanks,
I believe the change should fix the problem, I'll retry the same process for a final confirmation.
